docs/specification/draft/client/elicitation.mdx

```diff
@@ -151,139 +257,237 @@ To request information from a user, servers send an `elicitation/create` request
 }
 ```
 
-**Reject Response Example:**
+### URL Elicitation Requests
+
+URL elicitation enables servers to direct users to external URLs for out-of-band interactions that must not pass through the MCP client. This is essential for auth flows, payment processing, and other sensitive or secure operations.
+
+<Note>
+  **Important**: URL elicitation is *not* for authorizing the MCP client's
```

Comment:
Let's make sure that we use uniform terminology. If it's "URL mode elicitations," it would make it clear that it's about the mode and not just requesting a URL through elicitations.